streams/src/test/java/org/apache/kafka/streams/kstream/internals/KStreamImplTest.java

```diff
@@ -108,14 +108,14 @@ public boolean test(String key, Integer value) {
             public Integer apply(Integer value1, Integer value2) {
                 return value1 + value2;
             }
-        }, JoinWindows.of("join-0"), stringSerde, intSerde, intSerde);
+        }, JoinWindows.of("join-0", 1), stringSerde, intSerde, intSerde);
```

Comment:
Should we explain the magic number `1` here?  Why `1`?  Can it be any other positive number?  One way to clarify would be to use a "label" variable like `int ANY_POSITIVE_SIZE=1;`, then call `JoinWindows.of("join-0", ANY_POSITIVE_SIZE)`.